examples/falling-balls/iosApp/iosApp/ContentView.swift

```diff
@@ -2,20 +2,17 @@ import UIKit
 import SwiftUI
 import shared
 
-struct ComposeView: UIViewControllerRepresentable {
-    func makeUIViewController(context: Context) -> UIViewController {
-        Main_iosKt.MainViewController()
-    }
-
-    func updateUIViewController(_ uiViewController: UIViewController, context: Context) {}
-}
-
 struct ContentView: View {
     var body: some View {
         ComposeView()
-                .ignoresSafeArea(.keyboard) // Compose has own keyboard handler
+            .ignoresSafeArea(.keyboard) // Compose has own keyboard handler
     }
 }
 
+struct ComposeView: UIViewControllerRepresentable {
+    func makeUIViewController(context: Context) -> UIViewController {
+        Main_iosKt.MainViewController()
+    }
 
-
+    func updateUIViewController(_ uiViewController: UIViewController, context: Context) {}
+}
```

Comment:
Here I changed the order.
I think ContentView shoud be on top of the file. Because file name is also ContentView.swift